lib/matplotlib/ticker.py

```diff
@@ -1238,40 +1261,41 @@ def format_eng(self, num):
         u'-1.00 \N{GREEK SMALL LETTER MU}'
 
         `num` may be a numeric value or a string that can be converted
-        to a numeric value with the `decimal.Decimal` constructor.
+        to a numeric value with ``float(num)``.
```

Comment:
Do we actually have a use case for supporting this?  Could this behavior be deprecated?  (this is not strictly needed for the PR)